chain/x/supply/keeper.go

```diff
@@ -115,3 +115,31 @@ func (k WrappedSupplyKeeper) MintCoins(ctx sdk.Context, moduleName string, amt s
 
 	return nil
 }
+
+// MintCoinsToAcc mints coins from module to account
+func (k WrappedSupplyKeeper) MintCoinsToAcc(
+	ctx sdk.Context,
+	moduleName string,
+	recipient sdk.AccAddress,
+	amt sdk.Coins,
+) error {
+	moduleAcc := k.GetModuleAccount(ctx, moduleName)
+	if moduleAcc == nil {
+		panic(sdkerrors.Wrapf(sdkerrors.ErrUnknownAddress, "module account %s does not exist", moduleName))
+	}
+
+	if !moduleAcc.HasPermission(supply.Minter) {
+		panic(sdkerrors.Wrapf(sdkerrors.ErrUnauthorized, "module account %s does not have permissions to mint tokens", moduleName))
+	}
+
+	logger := k.Logger(ctx)
+	err := k.SendCoinsFromModuleToAccount(ctx, moduleName, recipient, amt)
```

Comment:
You need to actually change the values in mint pool when you send some money. (get, change, set) I also recommend to create a special function to perform this (check comsos-sdk/distribution FundCommunityPool method)